tools/flang2/docs/xflag.n

```diff
@@ -5505,6 +5505,8 @@ The -ffast-math command-line option is present.
 Disable fast math attribute on floating-point addition.
 .XB 0x04:
 Disable fast math attribute on floating-point division.
+.XB 0x1000:
+The -ffp-contract=[fast|on] command-line option is present.
```

Comment:
Lines 5505 to 5512 have been indented by one space for no apparent reason. Could you fix that?

Reply:
This is weird. It happens only in this github web interface. The original code, neither the patch file definitely doesn't have this indent. Please pull the repository and see the same lines in any text editor.

Reply:
Oh you are right. It seems to be a misleading rendering error on GitHub's part.